### ISSUE TITLE
Segmentation fault for divisors(1)

### ISSUE THREAD

**Issue by trizen** [danaj/Math-Prime-Util-GMP#8]
Hi,

There seems to be a minor issue in `Math::Prime::Util::GMP::divisors(n)` when `n=1`:

```perl
use 5.014;
use Math::Prime::Util::GMP qw(divisors);
say for divisors(1);
```

Output:
```
[1]    17417 segmentation fault (core dumped)  perl x.pl
```

**Comment by danaj** [comment]
Segmentation faults are not good at all.  One line fix in b02d89a plus a test.

**Comment by trizen** [comment]
Fix confirmed. Thank you for taking the time in fixing it. :+1:
